modules/ROOT/pages/autoresize.adoc

```diff
@@ -1,7 +1,7 @@
 = Autoresize plugin
 
 :title_nav: Autoresize
-:description_short: :description: Automatically resize TinyMCE to fit content. :keywords: height width max_height min_height autoresize_on_init autoresize_overflow_padding autoresize_overflow_padding
+:description_short: :description: Automatically resize TinyMCE to fit content. :keywords: height width max_height min_height autoresize_overflow_padding autoresize_overflow_padding
```

Comment:
```suggestion
:description_short: :description: Automatically resize TinyMCE to fit content.
:keywords: height width max_height min_height autoresize_overflow_padding autoresize_overflow_padding
```
Looks like there was some conversion issues here, so not related to your change but might as well fix it up now.

Reply:
Those two are removed in a [separated PR](https://github.com/tinymce/tinymce-docs/pull/2250)

Reply:
It should all be in a single PR...